pkg/server/container_create_windows.go

```diff
@@ -166,8 +171,28 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 
 	log.G(ctx).Debugf("Container %q spec: %#+v", id, spew.NewFormatter(spec))
 
-	snapshotterOpt := snapshots.WithLabels(config.Annotations)
+	// If the config field is specified, set the snapshotter label to reuse the pods
+	// scratch space. This only affects LCOW at the moment.
+	val, ok := config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.reuse-scratch"]
+	if rhcso.ShareScratch || (ok && val == "true") {
+		config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.reuse-scratch"] = "true"
+		_, ok := config.Annotations["containerd.io/snapshot/io.microsoft.owner.key"]
+		if !ok {
+			config.Annotations["containerd.io/snapshot/io.microsoft.owner.key"] = sandboxID
```

Comment:
Should we log here that we're falling back to use the sandboxID?

Reply:
Probably not for this. We're not going to document this annotation, it's just a means to an end to get this working for now. There's no checks that the ID they would pass in is in the same pod or any of the sort.